src/cdk/portal/portal-directives.ts

```diff
@@ -47,9 +48,8 @@ export class CdkPortal extends TemplatePortal<any> {
   exportAs: 'cdkPortalOutlet, cdkPortalHost',
   inputs: ['portal: cdkPortalOutlet']
 })
-export class CdkPortalOutlet extends BasePortalOutlet implements OnDestroy {
-  /** The attached portal. */
-  private _portal: Portal<any> | null = null;
+export class CdkPortalOutlet extends BasePortalOutlet implements OnInit, OnDestroy {
+  private _hasInitialized = false;
```

Comment:
Add description? It's not clear from the name whether this means the directive itself is initialized or something else (maybe `_isInitialized`?)